src/ScottPlot5/ScottPlot5/FontService.cs

```diff
@@ -65,4 +66,44 @@ private static SKTypeface GetDefaultSerifFont()
 
         return SKTypeface.Default;
     }
+
+    /// <summary>
+    /// Return the name of the font which will best display the given character.
+    /// This method helps identify the best fonts for displaying Chinese, Japanese, and Korean (CJK) characters.
+    /// </summary>
+    public static string DetectFont(char c)
+    {
+        // https://github.com/ScottPlot/ScottPlot/issues/2746
+        return SKFontManager.Default.MatchCharacter(c).FamilyName;
+    }
+
+    /// <summary>
+    /// Set the default font to the best one for displaying the given character.
+    /// This method helps display Chinese, Japanese, and Korean (CJK) characters.
+    /// </summary>
+    public static void SetDefaultFont(char c)
+    {
+        // https://github.com/ScottPlot/ScottPlot/issues/2746
+        DefaultFontName = DetectFont(c);
+    }
+
+    /// <summary>
+    /// Set the default font to the best one for displaying the given character.
+    /// This method helps display Chinese, Japanese, and Korean (CJK) characters.
+    /// </summary>
+    public static void SetDefaultFont(string fontName, bool throwIfNotFound = false)
+    {
```

Comment:
I think this should at least return `bool success` or something like that. I can understand not wanting to throw by default, but this current signature makes it pretty hard to tell if the method has failed, and this is a method that will likely fail often.

Consider that if a user wants to detect a failure to set the font (i.e. to attempt a fallback font) they need to do this:
```cs
try
{
    SetDefaultFont("Comic Sans MS", true);
}
catch (InvalidOperationException e) 
{
    SetDefaultFont("Fallback Font"); // Potentially multiple fallback fonts
}
```

This should probably look more like this:
```cs
if (!SetDefaultFont("Comic Sans MS"))
    SetDefaultFont("Fallback Font");
```
Or my personal preference:
```cs
SetDefaultFont("Comic Sans MS", fallbackList); // Perhaps this should even just be SetDefaultFont(fontList);
```

Also `InvalidOperationException` is probably the wrong exception, maybe `FileNotFoundException` or `NotSupportedException`. Or a custom exception class.

Reply:
I'm adding a `bool Exists(string fontName)` method to make this easier, then the user can choose how to implement their own fallback logic